### PR TITLE
Update Intimidate Interactions

### PR DESCRIPTION
**Pull request by PartMan7**
Added Speed boost due to Rattled being hit by Intimidate (source: https://www.smogon.com/forums/threads/sword-shield-battle-mechanics-research.3655528/post-8285401).
Edited desc and shortDesc of Rattled to include new interaction.

### CODE REVIEW

**Comment by PartMan7** [comment]
Sorry about that mix-up.

File: data/mods/gen7/abilities.js [hunk 1]
```diff
@@ -34,6 +35,35 @@ let BattleAbilities = {
...
+		shortDesc: "This Pokemon's Speed is raised 1 stage if hit by a Bug-, Dark-, or Ghost-type attack.",
+		onAfterDamage(damage, target, source, effect) {
+			if (effect && (effect.type === 'Dark' || effect.type === 'Bug' || effect.type === 'Ghost')) {
+				this.boost({spe: 1});
+			}
+		},
+		id: "rattled",
+		name: "Rattled",
+		rating: 1.5,
+		num: 155,
```
Comment (urkerab) [inline data/mods/gen7/abilities.js]: I guess this could have been inherited and then the onAfterBoost function nulled out?

Comment (TheImmortal) [reply data/mods/gen7/abilities.js]: Would have had to add the descs too.

Comment (urkerab) [reply data/mods/gen7/abilities.js]: The idea was that I was commenting on lines +53 to +61 as being the properties that would have been inherited.

**Pull request merged**